app/Http/Controllers/EmailController.php

```diff
@@ -0,0 +1,75 @@
+<?php
+
+namespace App\Http\Controllers;
+
+use App\Mail\EmailToStudent;
+use Illuminate\Http\Request;
+use Illuminate\Support\Facades\DB;
+use Illuminate\Support\Facades\Mail;
+
+class EmailController extends Controller
+{
+
+    public function __construct()
+    {
+        parent::__construct();
+      //  $this->middleware('permission:hr.view', ['except' => 'teacher']);
+    }
+
+    /**
+     * Display a listing of the resource.
+     */
+    public function index(Request $request)
+    {
+        $student = "";
+        if($request->get("student")){
+            $student = $request->get("student");
+        }
```

Comment:
This seems a little odd. Why do you assign an empty string to the $student variable ? I think you could add an early return here (if request does not have "student", abort and redirect and flash an error message) and then on line 34, do: `'preselect' => $request->get("student"),`

Reply:
where would I be redirecting to?